README.md

```diff
@@ -438,7 +438,7 @@ Run this command to verify the Intel&reg; QAT OpenSSL\* Engine is loaded
 correctly:
 
 ```text
-cd /path/to/openssl_install/bin
```

Comment:
/path/to/openssl_install refers to install directory where openssl is installed using prefix option during openssl build step. This path that you give in the engine configure command using "--with-openssl_install_dir=/path/to/openssl_install"

I think you are referring to OpenSSL Source dir, where Openssl resides in the apps dir.